pages/linux/false.md

```diff
@@ -0,0 +1,8 @@
+# false
+
+> Does nothing, unsucessfully.
```

Comment:
I wonder if this line is helpful. Removing this doesn't change much.

Reply:
I think keeping `true` (#2270) and `false` in the same format would be good.

Reply:
@pxgamer ...so remove this line/

Reply:
Yes. I think it makes sense to keep both pages in sync.